plugins/doc_fragments/modules.py

```diff
@@ -47,9 +47,9 @@ class ModuleDocFragment(object):
   timeout:
     description:
     - The socket level timeout in seconds.
+    - The default value is 30 seconds.
```

Comment:
It would be good to highlight the new capability of now having the option to add login domain in the inventory when httpapi is used (line 77):
```
login_domain:
    description:
    - The login domain name to use for authentication.
    - The default value is Local.
    - If the value is not specified in the task, the value of environment variable C(MSO_LOGIN_DOMAIN) will be used instead.
    - This value can be specified in the inventory using ansible_httpapi_login_domain when the connection is ansible.netcommon.httpapi
    type: str
```